src/vs/workbench/contrib/terminal/browser/media/terminal.css

```diff
@@ -522,3 +522,7 @@
 .monaco-workbench .xterm.terminal.hide {
 	visibility: hidden;
 }
+
+.monaco-workbench .tabs-list .action-label.codicon.codicon-debug-restart {
+	color: inherit !important;
+}
```

Comment:
There should be a generic version of the codicon so you don't need to do this. I think best practice is also to create a new one based off a base icon so that icon themes can change only that icon.